autoscale/src/main/java/com/sequenceiq/periscope/config/DatabaseConfig.java

```diff
@@ -92,7 +102,33 @@ private Properties jpaProperties() {
         properties.setProperty("hibernate.hbm2ddl.auto", hbm2ddlStrategy);
         properties.setProperty("hibernate.show_sql", "false");
         properties.setProperty("hibernate.dialect", "org.hibernate.dialect.PostgreSQLDialect");
+        properties.setProperty("hibernate.default_schema", StringUtils.isEmpty(dbSchemaName) ? "public" : dbSchemaName);
         return properties;
     }
 
+    private String createDbUrl() {
+        String url;
+        if (!StringUtils.isEmpty(dbSchemaName)) {
+            url = String.format("jdbc:postgresql://%s/%s?currentSchema=%s", databaseAddress, dbName, dbSchemaName);
+        } else {
+            url = String.format("jdbc:postgresql://%s/%s", databaseAddress, dbName);
```

Comment:
I suggest to replace in the whole project to replace jdbc:postgresql to a parametrized version to minimize database specific codes.